### PR TITLE
Implement Bully

### PR DESCRIPTION
**Pull request by nfaletra**
https://www.bg-wiki.com/bg/Bully

Left intimidate effect at a base 15% proc rate until more research is done on how multiple party members affects it. No gear modifies this ability as far as I know.

Retail message displays the debuff as "intimidated," but I'm using EFFECT_DOUBT as the id to track it on the server side in case EFFECT_INTIMIDATE is ever used for something else.

### CODE REVIEW

**Comment by takhlaq** [comment]
no bulli no bulli

File: scripts/globals/abilities/bully.lua [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+require("scripts/globals/status")
+-----------------------------------
+
+function onAbilityCheck(player,target,ability)
+    return 0,0
+end
+
+function onUseAbility(player,target,ability)
+    target:addStatusEffectEx(dsp.effect.DOUBT, dsp.effect.INTIMIDATE, 15, 0, 30)
+    return dsp.effect.INTIMIDATE;
```
Comment (takhlaq) [inline scripts/globals/abilities/bully.lua]: we dont semicolons anymore

**Comment by zynjec** [comment]
image

File: scripts/globals/abilities/bully.lua (hunk 1 above)
Comment (zynjec) [reply scripts/globals/abilities/bully.lua]: BULLY

**Comment by TeoTwawki** [comment]
@itsMaru its sending out emails for those dude

File: scripts/globals/abilities/bully.lua (hunk 1 above)
Comment (nfaletra) [reply scripts/globals/abilities/bully.lua]: :kill_me:

**Pull request merged**